TicTacToe.py

```diff
@@ -0,0 +1,209 @@
+from IPython.display import clear_output

```

Comment:
w calym skrypcie jest za duzo enterów :P

Reply:
Nie do konca rozumiem co robisz w tym skrypcie :p. Masa rzeczy sie powtarza, nazwy funkcji bywaja enigmatyczne (`users_token`?), pelno ifów i pętli. Generalnie spróbowałbym to uprościć. Mozna by sprobowac zrobic plansze za pomocą macierzy, wtedy odejdą te mało intuicyjne słowniki.